packages/launcher-app/src/config.ts

```diff
@@ -23,11 +23,17 @@ export const authenticationMode = getEnv(process.env.REACT_APP_AUTHENTICATION, '
   || (keycloakUrl ? 'keycloak' : 'no');
 export const isKeycloakMode = authenticationMode === 'keycloak';
 
-export const keycloakConfig = isKeycloakMode ? {
+export const config = isKeycloakMode ? {
   clientId: requireEnv(process.env.REACT_APP_KEYCLOAK_CLIENT_ID, 'keycloakClientId'),
   realm: requireEnv(process.env.REACT_APP_KEYCLOAK_REALM, 'keycloakRealm'),
   url: requireEnv(process.env.REACT_APP_KEYCLOAK_URL, 'keycloakUrl'),
-} as KeycloakConfig : undefined;
+} as KeycloakConfig : authenticationMode === 'openshift' ? {
```

Comment:
I guess having a function with `if` to return the `authConfig` would make sense, double ternary operator can be hard to read.